frontend/js/src/playlists/Playlist.tsx

```diff
@@ -546,7 +547,73 @@ export default class PlaylistPage extends React.Component<
     newAlert("danger", "Error", error.message);
   };
 
-  exportToSpotify = async () => {
+  exportToSpotify = async (
+    playlistId: string,
+    playlistTitle: string,
+    auth_token: string
+  ) => {
+    const { newAlert } = this.props;
+
+    const result = await this.APIService.exportPlaylistToSpotify(
+      auth_token,
+      playlistId
+    );
+    const { external_url } = await result.json();
+    newAlert(
+      "success",
+      "Playlist exported to Spotify",
+      <>
+        Successfully exported playlist:{" "}
+        <a href={external_url} target="_blank" rel="noopener noreferrer">
+          {external_url}
```

Comment:
I know this isn't something you've added or changed, but I wonder if we should use `{playlistTitle}` here as the link text instead of the url while we're in here making changes.
What's your opinion on that?

Reply:
I think that makes sense since the playlist will have the same name on Spotify. It reads as "Take me to 'MyPlaylist' on Spotify" instead of `https://open.spotify.com/playlist/<some id>`.